pulser-core/pulser/sequence/_seq_drawer.py

```diff
@@ -246,50 +242,48 @@ def gather_data(seq: pulser.sequence.Sequence, gather_output: bool) -> dict:
             eom_start_buffers,
             eom_end_buffers,
         )
-        if interp_pts:
-            data[ch].interp_pts = dict(interp_pts)
-    if hasattr(seq, "_measurement"):
-        data["measurement"] = seq._measurement
+
+    if sampled_seq._measurement is not None:
+        data["measurement"] = sampled_seq._measurement
     data["total_duration"] = total_duration
     return data
 
 
-def draw_sequence(
-    seq: pulser.sequence.Sequence,
+def _draw_channel_content(
+    sampled_seq: SequenceSamples,
+    register: Optional[BaseRegister] = None,
     sampling_rate: Optional[float] = None,
     draw_phase_area: bool = False,
-    draw_interp_pts: bool = True,
     draw_phase_shifts: bool = False,
-    draw_register: bool = False,
     draw_input: bool = True,
     draw_modulation: bool = False,
     draw_phase_curve: bool = False,
-) -> tuple[Figure | None, Figure]:
-    """Draws the entire sequence.
+    shown_duration: Optional[int] = None,
+) -> tuple[Figure | None, Figure, Any, dict]:
+    """Draws the computed data for a SequenceSamples.
 
     Args:
-        seq: The input sequence of operations on a device.
+        sampled_seq: The input sequence of operations on a device.
+        register: If present, draw the register before the pulse
+            sequence, with a visual indication (square halo) around the qubits
```

Comment:
register: If present, draw the register before the pulse
            sequence samples, with a visual indication (square halo) around the qubits